tests/testthat/_snaps/dm.md

```diff
@@ -108,8 +108,18 @@
     Code
       writeLines(conditionMessage(expect_error(dm(dm_for_flatten(),
       dm_for_filter_duckdb()))))
-    Output
-      All `dm` objects need to share the same `src`.
+    Condition
+      Warning in `dm_for_filter_duckdb()`:
+      restarting interrupted promise evaluation
+      Warning in `dm_for_filter_duckdb()`:
+      restarting interrupted promise evaluation
```

Comment:
I don't think this is right -- `dm_for_filter_duckdb()` should at least work, the error indicates that the insertion order still isn't quite right.